options_test.go

```diff
@@ -0,0 +1,40 @@
+package envbuilder_test
+
+import (
+	"testing"
+
+	"github.com/coder/envbuilder"
+	"github.com/coder/serpent"
+	"github.com/stretchr/testify/require"
+)
+
+type testEnvCase struct {
+	Env      string
+	EnvValue string
+	Expected interface{}
+	Actual   func() interface{}
+}
+
+func TestEnvOptions(t *testing.T) {
+	t.Setenv("SETUP_SCRIPT", "setup.sh")
+	t.Setenv("CACHE_TTL_DAYS", "7")
+	t.Setenv("IGNORE_PATHS", "/var,/tmp")
+	t.Setenv("SKIP_REBUILD", "true")
```

Comment:
The previous code was using https://pkg.go.dev/strconv#ParseBool, but I'm not sure what the new code is using so we might need to test `1`/`0`/`true`/`false` (and the latter two's capitalised equivalents).